swagger.yaml

```diff
@@ -3733,6 +3733,10 @@ components:
           type: string
           format: date-time
           description: Time when the survey status was marked as DONE, null if survey is not in DONE state
+        creatorName:
+          type: string
+          description: Name of the creator of the survey
+          readOnly: true
```

Comment:
Same comment as in API. Either change to "creatorDisplayName" or to "creatorFullName" OR keep the "creatorName" and add an extra parameter for "creatorSurname".

Reply:
I think creatorDisplayName is nice option indeed